fastly-config.vcl

```diff
@@ -57,6 +47,11 @@ sub vcl_deliver {
 	} else if (req.url ~ "^/v\d/polyfill\..*[\?\&]ua=" && req.http.X-Orig-URL && req.http.X-Orig-URL !~ "[\?\&]ua=") {
 		add resp.http.Vary = "User-Agent";
 	}
+
+	if (req.url ~ "[\&\?]rum=1") {
+    	add resp.http.Set-Cookie = "FastlyDC=" server.datacenter "; Path=/; HttpOnly; max-age=60";
```

Comment:
I think we should get qa on https, Next can't test our qa as their csp policy doesn't allow http resources.